File: JavaScript/ordenamientoBurbuja.js

let arreglo = [];
for (let i = 0; i < 10; i++) {
  arreglo.push(Math.floor(Math.random() * 101));
}

console.log("Arreglo antes de ordenar:");
console.log(arreglo.join(" "));


for (let i = 0; i < arreglo.length - 1; i++) {
  for (let j = 0; j < arreglo.length - 1 - i; j++) {
    if (arreglo[j] > arreglo[j + 1]) {

      let aux = arreglo[j];
      arreglo[j] = arreglo[j + 1];
      arreglo[j + 1] = aux;
    }
  }
}

console.log("Arreglo ordenado:");
console.log(arreglo.join(" "));

File: Python/ordenamientoBurbuja.py
import random


arreglo = [random.randint(0, 100) for _ in range(10)]

print("Arreglo antes de ordenar:")
print(" ".join(map(str, arreglo)))

for i in range(len(arreglo) - 1):
    for j in range(len(arreglo) - 1 - i):
        if arreglo[j] > arreglo[j + 1]:
            # Intercambiar valores
            arreglo[j], arreglo[j + 1] = arreglo[j + 1], arreglo[j]

print("Arreglo ordenado:")
print(" ".join(map(str, arreglo)))
